plugins/logitech-bulkcontroller/logitech-bulkcontroller.quirk

```diff
@@ -20,8 +20,3 @@ InstallDuration = 1500
 # Sight
 [LOGI\MODEL_VU0067]
 InstallDuration = 1500
-
-# Sight as Standalone device
-[USB\VID_046D&PID_087A]
-Plugin = logitech_bulkcontroller
-InstallDuration = 1500
```

Comment:
Deliberate? This is what I saw when I plugged it in directly.

Reply:
It seems Sight device is only supported as Pheripheral to RallyBar/RallBarMini. I am working with device team to determine if standalone is supported or not. 
I will update the PR accordingly

Reply:
Confirmed with Logitech device team, Sight not supported as Standalone device. So updated .quirk file accordingly

Reply:
@hughsie As for removing emulators for Sight device, do we just remove, logi-sight-setup.json, logi-sight.json files and remove Sight reference from logi-rally-bar.json file? 
I see that build is failing after removing support for Sight as standalone device in .quirk file.